pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/PillarboxCastPlayer.kt

```diff
@@ -119,20 +176,22 @@ class PillarboxCastPlayer(
     }
 
     override fun addListener(listener: Player.Listener) {
-        castPlayer.addListener(listener)
+        castPlayer.addListener(ForwardingListener(this, listener))
         listeners.add(listener)
     }
 
+    @SuppressLint("ImplicitSamInstance")
     override fun removeListener(listener: Player.Listener) {
-        castPlayer.removeListener(listener)
+        castPlayer.removeListener(ForwardingListener(this, listener))
         listeners.remove(listener)
     }
 
     override fun getAvailableCommands(): Player.Commands {
-        val isShuffleAvailable = remoteMediaClient?.mediaStatus?.isMediaCommandSupported(MediaStatus.COMMAND_QUEUE_SHUFFLE) == true
-
+        val isShuffleAvailable = getMediaStatus()?.isMediaCommandSupported(MediaStatus.COMMAND_QUEUE_SHUFFLE) == true
+        val availableCommands = getMediaStatus()?.isMediaCommandSupported(MediaStatus.COMMAND_EDIT_TRACKS) == true
```

Comment:
What about using a more precise name?
```suggestion
        val isEditTracksAvailable = getMediaStatus()?.isMediaCommandSupported(MediaStatus.COMMAND_EDIT_TRACKS) == true
```